src/js/ConfigInserter.js

```diff
@@ -1,107 +0,0 @@
-'use strict';
-
-const ConfigInserter = function () {
-};
-
-const CUSTOM_DEFAULTS_POINTER_ADDRESS = 0x08002800;
-const BLOCK_SIZE = 16384;
-
-function seek(firmware, address) {
-    let index = 0;
-    for (; index < firmware.data.length && address >= firmware.data[index].address + firmware.data[index].bytes; index++);
-
-    const result = {
-        lineIndex: index,
-    };
-
-    if (firmware.data[index] && address >= firmware.data[index].address) {
-        result.byteIndex = address - firmware.data[index].address;
-    }
-
-    return result;
-}
-
```

Comment:
Where these only used for old firmware flashing?

Reply:
The hexes get updated by the build API before downloading.